src/screens/CreateNFT.js

```diff
@@ -314,7 +314,7 @@ function CreateNFT() {
         <div className="row">
           <div className="form-group col-4">
             <label>{t`Amount`}</label>
-            <InputNumber required className="form-control" precision={0} placeholder="How many NFT units to create" onValueChange={onAmountChange} />
```

Comment:
Why did you remove the placeholder?

Reply:
Please see https://github.com/HathorNetwork/hathor-wallet/pull/699#discussion_r1869887972